cal-itp/benefits
# New Login.gov button design

close #653

## What this PR does
- Adds Public Sans font
- Uses Public Sans font for `Get started with` text in button
- Adjust sizes of buttons, alignment, line-height, font on Desktop and Mobile
 
## How to test
- Test Desktop, Mobile for the Elig Start page
- Test the Enrollment Success when Logged In and Logged Out pages (because this page also has a Login.gov button) - Ensure no regressions

## Button composition

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/173708988-f6fc68f8-efe5-4a04-9152-ac949c8f5a84.png">
<img width="449" alt="image" src="https://user-images.githubusercontent.com/3673236/173709007-d56bf0c9-6c3f-4890-bf5e-ecb52d4ed53d.png">
